### DISCUSSION TITLE
Entering the pre-paradigm phase of 'confusion'

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#191] [Metaphysics and Philosophy]
We now can find the same authors on contradicting papers, an example of the *"confusion that characterizes the pre-paradigm stage in which there are many competing hypotheses"* as described by Kuhn.

[This bigthink.com article](https://bigthink.com/hard-science/dark-matter-alternative-modified-newtonian-dynamics-ruled-out/) refers to the paper by ***I. Banik*** *et al.* [https://academic.oup.com/mnras/article/527/3/4573/7342478](https://academic.oup.com/mnras/article/527/3/4573/7342478), a paper which does not support MOND.

  Also published one day before, by Mazurenko, ***I. Banik***, Kroupa, and Haslbauer "A simultaneous solution to the Hubble tension and observed bulk flow within 250/h Mpc" [https://academic.oup.com/mnras/article/527/3/4388/7337338](https://academic.oup.com/mnras/article/527/3/4388/7337338), a paper that supports MOND as resolving two tensions in cosmology.

Of course, neither ΛCDM nor MOND will survive the shift...

**Comment by RedshiftDrift** [comment]
And others are taking advantage of the confusion to boast their own model
"Wide Binaries and Modified Gravity (MOG)", John W. Moffat, [https://arxiv.org/abs/2311.17130](https://arxiv.org/abs/2311.17130)

**Reply by bgreenman** [reply]
a great time to do research in the area of galaic gravity puzzles

**Reply by RedshiftDrift** [reply]
Sabine on MOND and Moffat gravity MOG
[https://www.youtube.com/watch?v=ERazd-SYk_s&t=418s](https://www.youtube.com/watch?v=ERazd-SYk_s&t=418s)

**Comment by RedshiftDrift** [comment]
"Discussion of Dark Matter and Modified Gravity" Simon White and Stacy McGaugh

If you don't have two hours to spare, watch the last 10 minutes https://youtu.be/h-p5dLQ4Fy8?t=6653
https://tritonstation.com/2024/01/08/discussion-of-dark-matter-and-modified-gravity/

  I am posting this, not to promote MOND, but for the same reason stated by McGaugh: "I hope it can provide an example for how it is possible to have a positive discussion about a controversial topic without personal animus."

**Reply by HanDeBruijn** [reply]
Related: [Checking in on Troubles with Dark Matter](https://github.com/orgs/a-cosmology-group/discussions/80#discussioncomment-6328995).

**Comment by RedshiftDrift** [comment]
The way forward in cosmology is to abandon ΛCDM, as explained here [https://www.youtube.com/watch?v=ro3GG0L9YPg&t=1668s](https://www.youtube.com/watch?v=ro3GG0L9YPg&t=1668s)

**Reply by Pierre314159** [reply]
The way forward for cosmology is to abandon the idea that there must be a single explanation for every observation and his dog. I find it very possible and plausible that there can be a cause for the cosmological redshift and a cause for the CMB and that both causes be different, in that the cause for the redshift does not explain the CMB and vice versa. There may be a cause for the rotation of galaxies that is different from the cause of the large scale structures, and inversely. Etc, etc..

That is where my reflection lead me after years of reading you folks on CMG, among others. I don’t believe the quest for a « model » that explains everything is a sound endeavour, lest we seek to be be equals of God…

**Reply by RedshiftDrift** [reply]
What about a cause for the redshift which:
1) in combination with Newtonian gravity, mechanics, plasma physics and electromagnetism, explains galaxy rotation?
2) in combination with optics, quantum mechanics and plasma physics, explains the CMB?
3) in combination with thermodynamics, plasma physics, special relativity and nucleosynthesis, explains large scale structures?
4) in combination with nuclear physics, optics, plasma physics, Newtonian mechanics, explains quasar brightness?

This does not depend on the idea of a single explanation for every observation, does it?

No single explanation fits every observed phenomena.  Physics is built up from several models that interact with each other to explain the complexity of the world around us.  When we can't understand observations, the difficulty is to find how to change something without breaking everything else.  This will certainly not be done by someone not familiar with Newtonian mechanics, gravity, plasma physics, electromagnetism, optics, quantum mechanics, thermodynamics, special/general relativity, nucleosynthesis, nuclear physics, and optics.

**Reply by Pierre314159** [reply]
Yes. We are basically saying the same thing. Each one of your redshift, 1, 2, 3 and 4 is an explanation for one phenomenon, and each one ALONE does not explain any of the four others. 

But this is not what cosmologists do, is it? You are doing physics. Cosmology is the search for a "model" that will explain all of redshift, CMB, large scale structures, brightness of quasars, time lag of SN1a, abundance of Li, rotation of galaxies, etc, etc... Your approach is not the quest for a "model": it is five different solutions to five different observations. This is essentially what I'm saying. Any alternative "model" will get mainstream's objection nuber 867: "Your 'model' does not explain this or that...", one of the fallacies they use to dismiss hypotheses... 

It's been agreed (I think) among ACG that "The Universe", as a physical object, does not and cannot exist. "Cosmology", therefore, is the search for a model of something that "exists" solely as an abstraction ([Merriam-Webster](https://www.merriam-webster.com/dictionary/cosmology) defines Cosmology as a branch of metaphysics...). That's why I'm saying that the quest for a "model" of the "Universe" is futile... It doesn't even qualify as science... The Big Bang Theory needs to be ditched, not replaced.

> how to change something 

I wouldn't call STr a "change"; rather an addition.

**Reply by HanDeBruijn** [reply]
@Pierre314159 writes:
>I don’t believe the quest for a « model » that explains everything is a sound endeavour

Right !! Here is a quote from the [Cosmology main page](http://www.alternatievewiskunde.nl/cosmology/wild.htm) on my website.
>Arp's hypothesis is shown to be compatible with a static Euclidean universe, which is eternal in "atomic time" and has moments of creation (i.e. one or more beginnings) in "orbital time". In this model, _gravitation is completely separated from all other forces in nature_; it is e.g. distinct from quantum mechanics and electrodynamics

Meaning that a [Grand Unified Theory](https://en.wikipedia.org/wiki/Grand_Unified_Theory) is definitely beyond our scope. Concerning [Classical unified field theories](https://en.wikipedia.org/wiki/Classical_unified_field_theories) Wikipedia too admits that, finally
>none have been generally accepted by physicists yet.

**Reply by RedshiftDrift** [reply]
@Pierre314159 
> [Merriam-Webster](https://www.merriam-webster.com/dictionary/cosmology) defines Cosmology as a branch of metaphysics...

... where it belongs.  We basically agree, applying "relativity, gravity, thermodynamics..." to the entire universe is nonsense since these theories are valid for phenomena inside the universe.

Indeed, what I do is astrophysics and show that our laws of physics are incompatible with ΛCDM.  MOND, MOG and all these gravity theories have to be tested from observations.  My support for Kroupa, McGaugh is not for their cosmology model, but for the astrophysical observations they make to show that ΛCDM is wrong.
- MOND combined with galaxy dynamics explains rotation curves,
- MOND and plasma physics explain the collision of the 'Bullet cluster',
- MOND and stellar physics explains the 'wide binaries',
- MOND explains the CMB multipoles, etc.

Unfortunately, Kroupa puts his "cosmlogist's hat" and tries to build a cosmology (the BMoC), but McGaugh doesn't often go that far.

Scientifically, physical theories cannot be used to give us a model of the universe.  Not because these are beyond our scope, but because the universe is a metaphysical concept that is not dealt with by science.  (I tell Eric Lerner that "The Big Bang never happen", the title of his book, is a metaphysical statement, not a scientific statement.)

**Reply by HanDeBruijn** [reply]
If we adhere to the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs), then _Intrinsic Red-Shift_ equals the cosmological redshift as observed.
Intrinsic redshift is completely described by (classical) quantum mechanics and nothing else.
The Variable Mass in that explanation is of course _inertial_. But we know that _inertial mass = gravitational mass_.
As a consequence, (intrinsic) cosmological redshift is coupled with gravity via the variable mass.
